templates/flavors/azure-cni-v1/patches/kubeadm-control-plane.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+kind: KubeadmControlPlane
+metadata:
+  name: ${CLUSTER_NAME}-control-plane
+  namespace: default
+spec:
+  kubeadmConfigSpec:
+    initConfiguration:
+      nodeRegistration:
+        kubeletExtraArgs:
+          max-pods: "110"
+          # non-masquerade-cidr: "0.0.0.0/0" # this is not a default property, can avoid setting it
```

Comment:
if this a TODO, let's either fix it or remove it

Reply:
Removing it. It was more of an information for my testing.